src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -191,11 +191,11 @@ void GcsPlacementGroupScheduler::PrepareResources(
         auto result = reply.success() ? Status::OK()
                                       : Status::IOError("Failed to reserve resource");
         if (result.ok()) {
-          RAY_LOG(DEBUG) << "Finished leasing resource from " << node_id
-                         << " for bundles: " << GetDebugStringForBundles(bundles);
+          RAY_LOG(INFO) << "Finished leasing resource from " << node_id
+                        << " for bundles: " << GetDebugStringForBundles(bundles);
         } else {
-          RAY_LOG(DEBUG) << "Failed to lease resource from " << node_id
-                         << " for bundles: " << GetDebugStringForBundles(bundles);
+          RAY_LOG(WARNING) << "Failed to lease resource from " << node_id
```

Comment:
Can we change this to INFO since this is not a real issue

Reply:
I think the WARNING log level would be better as it can serve as a good indication of potential errors.

Reply:
I still think it should be INFO given this is expected to happen and is normal.

```
WARN – the log level that indicates that something unexpected happened in the application, a problem, or a situation that might disturb one of the processes. But that doesn't mean that the application failed. The WARN level should be used in situations that are unexpected, but the code can continue the work.
```

Reply:
Sure. Also fine. INFO works too.